lib/src/vector/operator/statistics.dart

```diff
@@ -0,0 +1,37 @@
+import '../vector.dart';
+
+extension StatisticsExtension<T> on Vector<T> {
+  /// Returns the index of the minimum element.
+  int minimumIndex() {
```

Comment:
Isn't this already available on `vector.iterable`?

Reply:
I removed the file and use `iterable.reduce(max)` instead of `maximum()`. Thanks.